lib/litani.py

```diff
@@ -168,6 +176,7 @@ def cache_pointer_dirs():
                 possible_cache = pathlib.Path(pointer)
                 if possible_cache.exists():
                     logging.debug("cache is at %s", possible_cache)
+                    CacheDir.path = possible_cache
```

Comment:
Isn't the following more typical of an instantiation?

```python
cache_dir = CacheDir(path=possible_cache)
```

In fact, can't you just say `return CacheDir(path=possible_cache) and remove both this line and the line beneath it?

Reply:
Right. Doesn't the following return the path attribute?

`return CacheDir(path=possible_cache).path`

Reply:
Yes. Made some changes along with this.